### PR TITLE
Istio overview page, no Grafana link yet

### PR DESCRIPTION
**Pull request by mantis-toboggan-md**
#1084 #1132 
Screen Shot 2020-09-01 at 10 34 54 AM

### CODE REVIEW

**Comment by deniseschannon** [comment]
@brendarearden how does this screenshot look to you?

**Comment by deniseschannon** [comment]
@lvuch how does this look design wise to you?

**Comment by lvuch** [comment]
There are some minor styling changes I'd like to see. I can make them so they apply to all pages. The logo and external link seem too large so if you reduce that size that'd be great. In the configure box inline code excerpts should have some style beyond just mono type (I can do this once the pr is merged).

**Comment by brendarearden** [comment]
screen shot looks like I was expecting 👍

**Review by brendarearden — APPROVED** [review]

File: components/Glance.vue
Comment (vincent99) [inline components/Glance.vue:10]: Not going to hold up the rest of the PR at this point, but I don't really love having to declare the slots you want and then say them again to use them (and hope they match) as an API.

Tabbed/Tab does this automatically by [providing](https://github.com/rancher/dashboard/blob/master/components/Tabbed/index.vue#L49-L67), [injecting](https://github.com/rancher/dashboard/blob/master/components/Tabbed/Tab.vue#L3), and [calling](https://github.com/rancher/dashboard/blob/master/components/Tabbed/Tab.vue#L37-L43) hooks when the components are added/removed.

If they don't need to handle dynamical add/remove, then just using $children on the glance (like tabbed [used to](https://github.com/rancher/dashboard/commit/e9c2f0e942c35ee2e3e2bd49ee782fa34b901037#diff-5c8b1e033310adfda729578fd1282e1d) do) also works and is simpler.

**Pull request merged**
